pkgs/development/ocaml-modules/tezos-rust-libs/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitLab, topkg, findlib, cargo, rustc, opaline }:
+
+stdenv.mkDerivation rec {
+  pname = "tezos-rust-libs";
+  version = "1.0";
+
+  src = fetchFromGitLab {
+    owner = "tezos";
+    repo = "tezos-rust-libs";
+    rev = "v${version}";
+    sha256 = "1ffkzbvb0ls4wk9205g3xh2c26cmwnl68x43gh6dm9z4xsic94v5";
+  };
+
+  buildInputs = [
+    topkg
+    findlib
+  ];
+
+  propagatedBuildInputs = [ cargo rustc ];
+
+  buildPhase = ''
+    mkdir .cargo
+    mv cargo-config .cargo/config
+    cargo build --target-dir target --release
+  '';
+
+  installPhase = ''
+    ${opaline}/bin/opaline -prefix $out -libdir $OCAMLFIND_DESTDIR/lib -name tezos-rust-libs
+  '';
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://gitlab.com/tezos/tezos-rust-libs";
+    description = "Tezos: all rust dependencies and their dependencies";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.ulrikstrid ];
+  };
+
+  createFindlibDestdir = true;
```

Comment:
Is this needed?

Reply:
It seems like this line is not needed, but the install fails if I don't have findlib as a dependency